app/assets/stylesheets/application.scss

```diff
@@ -85,6 +86,7 @@ $unpaid: #B00100;
       -border-radius: 10px;
       border-radius: 10px;
       background-color: $paleGray;
+      color: $blue;
```

Comment:
If this has the same color as the parent li, the color doesn't _have_ to be specified here.